src/index.ts

```diff
@@ -12,6 +12,7 @@ export async function renderScreenshot(options: Partial<TerminalScreenshotOption
 
   const templatePath = await generateTemplate(validatedOtions);
   const browser = await puppeteer.launch({
+    headless: "new",
```

Comment:
is that needed/left over from #10?

```suggestion
```

Reply:
Nope, it's a way to mute this Puppeteer warning:
>       Puppeteer old Headless deprecation warning:                                                                                                                                                    
>         In the near future `headless: true` will default to the new Headless mode                                                                                                                    
>         for Chrome instead of the old Headless implementation. For more                                                                                                                              
>         information, please see https://developer.chrome.com/articles/new-headless/.                                                                                                                 
>         Consider opting in early by passing `headless: "new"` to `puppeteer.launch()`                                                                                                                
>         If you encounter any bugs, please report them to https://github.com/puppeteer/puppeteer/issues/new/choose.